_sass/blocks/explore/_regions-list.scss

```diff
@@ -83,13 +83,36 @@
     }
   }
 
-  tr > *:first-child {
-    padding-left: $base-padding;
-    width: 80%;
+  tr > *:last-child {
+
   }
 
-  tr > *:last-child {
+  tr {
+    > *:first-child {
+      padding-left: $base-padding;
+      width: 80%;
+
+      &.narrow {
+        width: 50%;
+      }
+    }
+
+    .variance,
+    .centered {
+      text-align: center;
+    }
 
+    .variance {
+      vertical-align: middle;
+    }
+  }
+
+  tr > td > .reportee {
```

Comment:
Merge rule `tr > td > .reportee` with rule on line 90<br>Selector should have depth of applicability no greater than 3, but was 4